src/chrome/chromeDebugAdapter.ts

```diff
@@ -945,8 +945,8 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         }
 
         const committedBps = this._committedBreakpointsByUrl.get(script.url) || [];
-        if (committedBps.indexOf(params.breakpointId) === -1) {
-            committedBps.push(params.breakpointId);
+        if (committedBps.findIndex(committedBp => committedBp.breakpointId === params.breakpointId) === -1) {
```

Comment:
I prefer `if (!committedBps.find(...` instead of checking index -1

Reply:
changed